src/components/CommentList.jsx

```diff
@@ -0,0 +1,20 @@
+import { Card, Heading, Stack } from "@chakra-ui/react";
+
+function CommentList({ comments }) {
+  return comments.map((comment, i) => (
+    <div key={i}>
```

Comment:
Would be useful to see the usernames here too as it's not visible who authored the comment (will be hard to identify your own comments when you get around to posting them for example)